### ISSUE TITLE
Conditionally activate fitBounds effect

### ISSUE THREAD

**Issue by rodrigofcaba** [dudesuh/react-leaflet-multi-options-polyline#37]
Hello again Ricardo,

I am not quite sure if this should be more of a pull request than an issue. I have a simple question though. If this should be a pull request, please, say so and I will try to suggest one. Probably you are faster implementing this if you agree on its usefulness.

I don't know if the prop "zoomAnimation" is intended to avoid the map fitting bounds of the polyline every time. If that is the case, it doesn't work. If that is not what it does, please, can you explain what this prop is used for?

And my suggestion is precisely to implement a way  (maybe a (boolean) prop) to  avoid fitting bounds of polyline each time the component rerenders. That's because, for instance, in my use case, the line renders when a new point flown by an aircraft is pushed to the state. However, due to this behavior, I am not able to use map zoom anymore. As long as the tracker receives new data (each second), the map fits polyline bounds and I cannot use zoom. 

A way to tell the polyline not to force the map to fit its bounds would be nice. If it could be usefull in general, I can make a pull request or you could implement it. Otherwise, if this behavior is too specific for my use case, I can try to implement it myself. I guess it should be "as easy as" including a conditional cheking the prop suggested here:

```javascript
useEffect(() => {
        const bounds = getBounds(props.positions);
        map.fitBounds(
            new L.LatLngBounds(
                new L.LatLng(bounds.maxLat, bounds.minLng),
                new L.LatLng(bounds.minLat, bounds.maxLng)
            )
        );
    })
```

Thanks in advance!

**Comment by dudesuh** [comment]
Hi Rodrigo,

There was a bug where the component would call fitBounds every time it updated, but i already corrected in v3.0.10.
Update to this new version and test it again.

**Comment by rodrigofcaba** [comment]
Hi Ricardo,

Now everything is working fine. 

Thanks once more!
